src/module-mentions.js

```diff
@@ -2,9 +2,10 @@ const h = (tag, attrs, ...children) => {
   const elem = document.createElement(tag);
   Object.keys(attrs).forEach(key => elem[key] = attrs[key]);
   children.forEach(child => {
-    if (typeof child === "string")
+    if (typeof child === "string") {
       child = document.createTextNode(child);
-      elem.appendChild(child);
+    }
+    elem.appendChild(child);
   });
```

Comment:
This section above needs some review. From now on we should explicitly say what is included our if statements. Just in case we make a mistake when coding.

Also this line gives me errors in the demo page.

Can either one of you review this issue? @satheesh12 @yeminhtut